TestApp/TestConfigurationApplication.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
```

Comment:
Need to add `<IsPackable>false</IsPackable>` here so we don't accidentally ship this as a NuGet package during the `build.cmd NuGet` stage.